src/tests/routes/controllers/users.test.js

```diff
@@ -0,0 +1,40 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+
+import app from '../../../server';
+
+chai.use(chaiHttp);
+const { expect } = chai;
+
+const usersUrl = '/api/v1/users';
+const profileUrl = '/api/v1/profiles/gentlejane';
+
+describe('List Users functionality', () => {
+  context('GET all users', () => {
+    it('should fetch all authors', async () => {
+      const res = await chai.request(app).get(usersUrl);
+      expect(res.status).to.equal(200);
+      expect(res.body.message)
+        .to.be.a('String')
+        .to.eql('success');
+      expect(res.body.data[0])
+        .to.be.an('object')
+        .to.have.property('email')
+        .to.eql('igbominadeveloper@ah.com');
+    });
+  });
+
+  context('GET user profile', () => {
```

Comment:
Nice work here, but I think you could also add test for when an author does not exist.

Reply:
Feedback implemented.